Mathlib/MeasureTheory/Measure/OuterMeasure.lean

```diff
@@ -681,7 +700,7 @@ theorem ofFunction_le (s : Set α) : OuterMeasure.ofFunction m m_empty s ≤ m s
   let f : ℕ → Set α := fun i => Nat.casesOn i s fun _ => ∅
   infᵢ_le_of_le f <|
     infᵢ_le_of_le (subset_unionᵢ f 0) <|
-      le_of_eq <| tsum_eq_single 0 <| by rintro (_ | i) <;> simp [f, m_empty]
+      le_of_eq <| tsum_eq_single 0 <| by rintro (_ | i); simp; simp [m_empty]
```

Comment:
If you want to deal with two subgoals separately, they should be on their own line with a cdot in front